src/main/java/org/metricshub/jawk/jrt/AssocArray.java

```diff
@@ -168,13 +164,10 @@ public boolean isIn(Object key) {
 			return true;
 		}
 
-		try {
```

Comment:
Please revert all logic changes, as it breaks an additional BWK.t test (6 failures instead of 5 before the change).

Simply remove the calls to LOG.debug() which are totally unnecessary in these cases. (The exception is a valid code path, and not really an error).